src/masonite/drivers/session/RedisDriver.py

```diff
@@ -81,25 +86,25 @@ def save(
         for key in deleted_flashed:
             self.delete(f"{self.get_flash_prefix()}{key}")
 
-    def get_session_namespace(self, session_id=None):
+    def get_session_namespace(self, session_id=None) -> str:
```

Comment:
```suggestion
    def get_session_namespace(self, session_id: str = None) -> str:
```

Reply:
Aah yes I see what you mean ok ....  Done